hydrus/samples/doc_writer_sample.py

```diff
@@ -58,17 +71,46 @@
                    op_returns,
                    op_status)
 
-
-"""Add the operation to the Class"""
+"""Same way add DELETE, PUT and GET operations"""
```

Comment:
@vddesai1871  Looks good! Could you please replace the `"""` with simple comments.